cidc_api/shared/gcloud_client.py

```diff
@@ -133,26 +135,25 @@ def grant_download_access(user_email: str, trial_id: str, upload_type: str):
     policy = bucket.get_iam_policy(requested_policy_version=3)
     policy.version = 3
 
-    # check if a binding for this user/trial/upload_type combo already exists
-    binding_exists = False
-    for binding in policy.bindings:
-        if _is_matching_conditional_binding(binding, member_id, prefix_expression):
-            binding_exists = True
-            break
-
-    # if the binding doesn't exist, create it
-    if not binding_exists:
-        policy.bindings.append(
-            {
-                "role": GOOGLE_DOWNLOAD_ROLE,
-                "members": {member_id},
-                "condition": {
-                    "title": f"Download access on {url_prefix}* for {user_email}",
-                    "description": f"Auto-generated by the CIDC API on {datetime.datetime.now()}",
-                    "expression": f"{prefix_expression} && {ttl_expression}",
-                },
-            }
+    # find the policy binding for this user if one exists
+    binding_index, binding = _get_user_download_binding(policy, user_email)
+
+    # add the permission clause in the binding condition expression if it isn't already included
+    expression = binding["condition"]["expression"]
+    if prefix_expression not in expression:
+        full_clause = f"({prefix_expression} && {ttl_expression})"
+        expression = (
+            f"{expression}{_DOWNLOAD_CLAUSE_OR}{full_clause}"
```

Comment:
not sure but maybe like this? 
```suggestion
            f"({expression}){_DOWNLOAD_CLAUSE_OR}{full_clause}"
```